pkg/sqlmigrations/migrations.go

```diff
@@ -885,20 +826,37 @@ func addRootToAdminRole(ctx context.Context, r runner) error {
 //
 // TODO(benesch): Remove this migration in v2.1.
 func upgradeTableDescsToInterleavedFormatVersion(ctx context.Context, r runner) error {
-	return upgradeTableDescsWithFn(ctx, r, func(desc *sqlbase.TableDescriptor) (bool, error) {
-		return desc.MaybeUpgradeFormatVersion(), nil
-	})
+	tableDescFn := func(desc *sqlbase.TableDescriptor) (bool, error) {
+		return desc.MaybeFillInDescriptor(), nil
+	}
+	return upgradeDescsWithFn(ctx, r, tableDescFn, nil)
+}
+
+// ensureAdminRolePrivileges ensures that all descriptors have privileges
+// for the admin role.
+//
+// TODO(mberhault): Remove this migration in v2.1.
+func ensureAdminRolePrivileges(ctx context.Context, r runner) error {
+	tableDescFn := func(desc *sqlbase.TableDescriptor) (bool, error) {
+		return desc.Privileges.MaybeAddAdminPrivileges(desc.ID), nil
```

Comment:
This is technically taken care of by `grantAdminPrivileges` before this change, and by `upgradeTableDescsToInterleavedFormatVersion`, but it doesn't hurt to specify it again.